src/hal/arm/exceptions.c

```diff
@@ -57,8 +57,11 @@ static int exceptions_i2s(char *prefix, char *s, unsigned int i, unsigned char b
 	char c;
 	unsigned int l, k, m;
 
-	m = hal_strlen(prefix);
-	hal_memcpy(s, prefix, m);
+	m = 0;
+	while (*prefix) {
+		s[m] = *prefix++;
+		m++;
+	}
```

Comment:
Maybe _for_ loop fits here better? 
`for (m = 0; *prefix != '\0'; ++m)`

Reply:
This code has been accepted in commit f46531644d938fb6fbe0fe730ee29203340e2573
but I can change it,
the code above and e.g.
```
for (m = 0; prefix[m] != 0; ++m)
    s[m] = prefix[m];
```
are the same ( generated by compiler )

Reply:
I myself prefer _for_ version better ;)